INSTALL.md

```diff
@@ -81,12 +81,23 @@ directories or build directory with this script.
     $ ./configure --help
     Usage: ./configure [<options>]
 
-      --help             print this message
-      --prefix=<DIR>     set install root dir as <DIR>        (default: /usr/local)
-      --bindir=<DIR>     set executable install dir as <DIR>  (default: ${prefix}/bin)
-      --libdir=<DIR>     set library install dir as <DIR>     (default: ${prefix}/lib)
-      --mandir=<DIR>     set manual doc install dir as <DIR>  (default: ${prefix}/share/man)
-      --objdir=<DIR>     set build dir as <DIR>               (default: ${PWD})
+      --help                print this message
+      --prefix=<DIR>        set install root dir as <DIR>        (default: /usr/local)
+      --bindir=<DIR>        set executable install dir as <DIR>  (default: \${prefix}/bin)
+      --libdir=<DIR>        set library install dir as <DIR>     (default: \${prefix}/lib)
+      --mandir=<DIR>        set manual doc install dir as <DIR>  (default: \${prefix}/share/man)
+      --objdir=<DIR>        set build dir as <DIR>               (default: \${PWD})
+      --sysconfdir=<DIR>    override the etc dir as <DIR>
+      --with-elfutils=<DIR> search for elfutils in <DIR>/include and <DIR>/lib
+
+      -p                    preserve old setting
+
+      Some influential environment variables:
+        ARCH          Target architecture    e.g. arm, aarch64, or x86_64
+        CROSS_COMPILE Specify the compiler prefix during compilation
+                      e.g. CC is overridden by $(CROSS_COMPILE)gcc
+        CFLAGS        C compiler flags
+        LDFLAGS       linker flags
```

Comment:
Could you please add one more space between variable name and decription?  Same for the previous commit too.